core/storage/statistics/gae_models.py

```diff
@@ -674,6 +858,15 @@ def insert_submitted_answers(
         is interrupted and retried. Furthermore, this method may fail with a
         DeadlineExceededError if too many answers are attempted for submission
         simultaneously.
+
+        Args:
+            exploration_id: str. ID of the exploration currently being played.
+            exploration_version: int. Version of exploration.
+            state_name: str. Name of current state.
+            interaction_id: str. It tells which type of computations are
```

Comment:
Doc string is wrong. Use "ID of the interaction"

Reply:
done